templates/http/base/rhoai/notebook.yaml

```diff
@@ -0,0 +1,115 @@
+apiVersion: kubeflow.org/v1
+kind: Notebook
+metadata:
+  annotations:
+    notebooks.opendatahub.io/inject-oauth: 'true'
+    opendatahub.io/image-display-name: Minimal Python
+    openshift.io/display-name: {{values.name}}
+  name: {{values.name}}
+  namespace: {{values.dsp}}
+  labels:
+    app: {{values.name}}
+    opendatahub.io/dashboard: 'true'
+    opendatahub.io/odh-managed: 'true'
+spec:
+  template:
+    spec:
+      containers:
+        - resources:
+            limits:
+              cpu: '2'
+              memory: 8Gi
+            requests:
+              cpu: '1'
+              memory: 8Gi
+          readinessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /notebook/{{values.dsp}}/{{values.name}}/api
+              port: notebook-port
+              scheme: HTTP
+            initialDelaySeconds: 10
+            periodSeconds: 5
+            successThreshold: 1
+            timeoutSeconds: 1
+          name: {{values.name}}
+          livenessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /notebook/{{values.dsp}}/{{values.name}}/api
+              port: notebook-port
+              scheme: HTTP
+            initialDelaySeconds: 10
+            periodSeconds: 5
+            successThreshold: 1
+            timeoutSeconds: 1
+          env:
+            - name: NOTEBOOK_ARGS
+            - name: JUPYTER_IMAGE
+              value: 'image-registry.openshift-image-registry.svc:5000/redhat-ods-applications/s2i-minimal-notebook:2024.1'
+            - name: MODEL_ENDPOINT
+              value: http://{{values.name}}-model-server:{{values.modelServicePort}}
```

Comment:
QQ: the model pod deployed via `http/base/deployment-model-server.yaml` is in a different namespace. is the endpoint still valid and be able to connect?

Reply:
Hmm, that was the one thing I still needed to test today. Currently, the URL present wouldn't work. However, we can access it with the following URL:
```
<service-name>.<namespace>.svc.cluster.local
```

But i still need to figure out how to programmatically get the namespace from the template. Would `{{values.namespace}}` work?

Reply:
yes, `{{values.namespace}}` is the namespace deploys the model server.

Reply:
so it should be  `{{values.name}}-model-server.{{values.namespace}}.svc.cluster.local:{{values.modelServicePort}}`?

Reply:
Nice, I’ll go update the template then

Reply:
Updated